static/app/components/events/eventStatisticalDetector/lineChart.tsx

```diff
@@ -22,85 +22,98 @@ interface ChartProps {
   percentileSeries: Series[];
 }
 
-function LineChart({datetime, percentileSeries, evidenceData}: ChartProps) {
-  const theme = useTheme();
-  const router = useRouter();
+export type EventBreakpointChartData = ChartProps;
```

Comment:
just rename Chart props to EventBreakpointChartData

Reply:
I actually don't need datetime now that we're not generating zoom properties. So they won't be equal. Let me update that.